### ISSUE TITLE
No rule detects text after a closing parenthesis inside a block (fatal cmd syntax error)

### ISSUE THREAD

**Issue by grinser** [tboy1337/Blinter#36]
## Summary

Blinter does not report a fatal `cmd.exe` syntax error. Inside a parenthesized
block, a line can contain a group of parentheses. If text follows the closing
parenthesis, `cmd.exe` stops and the script returns exit code 255.

## Reproduction

```bat
@echo off
for %%D in (C) do (
    echo --- text (parens) ---
)
echo END-OK
exit /b 0
```

## Actual

`cmd.exe` stops at the `for` block and never prints `END-OK`:

```
"---" was unexpected at this time.
```

Exit code 255.

Blinter reports no error:

```
Line 1: BAT extension used instead of CMD for newer Windows (S007)
```

## Expected

An ERROR-level finding. The message should tell the user to escape the
parentheses with a caret, or to remove the text after the closing parenthesis.

## Root cause (suspected)

The trigger is text **after** the closing parenthesis, not the parentheses
alone. This works, because nothing follows the group:

```bat
for %%D in (C) do (
    echo text (parens)
)
```

This fails:

```bat
for %%D in (C) do (
    echo --- text (parens) ---
)
```

Inside a block, `cmd.exe` uses the closing parenthesis to end a command group.
It then reads the following text as a new command. `---` is not a command.

The same failure occurs one level deeper:

```bat
for %%F in (report.log) do (
    if exist %%F (
      echo --- %%F (found) ---
    )
)
```

## Environment

- Blinter 1.1.19
- Python 3.14.5
- Windows 11 25H2 (build 26100, WinPE, German)

The error text above is the English form. It was observed as
`"---" kann syntaktisch an dieser Stelle nicht verarbeitet werden.`

**Comment by tboy1337** [comment]
Hi there, thanks for the report, it should be fixed now in 1.1.21, if there are any more issues please let me know.
